src/exec_cmd.rs

```diff
@@ -18,7 +19,11 @@ pub fn run(command: &str, command_args: &[String]) {
                 ))
                 .to_owned()
         })
-        .or_else(|| command::find_system_bin(command));
+        .or_else(|| {
+            let path = env::var("PATH").expect("env var PATH is not defined");
+            let current_exe = env::current_exe().unwrap();
```

Comment:
Originally, I made the `current_exe` and `system_path` args so that I could test `find_system_bin()`. The problem is that I haven't really been able to test `find_system_bin()` with this, so what's the point of the new args now.